packages/core/src/utils/spanUtils.ts

```diff
@@ -147,20 +150,23 @@ export function spanToJSON(span: Span): SpanJSON {
   if (spanIsOpenTelemetrySdkTraceBaseSpan(span)) {
     const { attributes, startTime, name, endTime, parentSpanId, status, links } = span;
 
-    return dropUndefinedKeys({
-      span_id,
-      trace_id,
-      data: attributes,
-      description: name,
-      parent_span_id: parentSpanId,
-      start_timestamp: spanTimeInputToSeconds(startTime),
-      // This is [0,0] by default in OTEL, in which case we want to interpret this as no end time
-      timestamp: spanTimeInputToSeconds(endTime) || undefined,
-      status: getStatusMessage(status),
-      op: attributes[SEMANTIC_ATTRIBUTE_SENTRY_OP],
-      origin: attributes[SEMANTIC_ATTRIBUTE_SENTRY_ORIGIN] as SpanOrigin | undefined,
-      links: convertSpanLinksForEnvelope(links),
-    });
+    return dropUndefinedKeys(
```

Comment:
Do we even need to call `dropUndefinedKeys` here?

Reply:
<img width="742" alt="image" src="https://github.com/user-attachments/assets/592792c8-4b4c-4f2e-a598-76f0f9a9e462" />

Reply:
I extracted this out here: https://github.com/getsentry/sentry-javascript/pull/15792 - I think you are right 🤔